### PR TITLE
Clarify readme

### PR DESCRIPTION
**Pull request by empeje**
Fixes 

Todos:

- [x] Beautify Readme (#2238, [click here for live preview](https://github.com/open-learning-exchange/planet/blob/one-click-installation/README.md)) 
- [x] Seamless Dev Setup (#2236)
- [ ] One click installation (#2235)

### CODE REVIEW

**Comment by empeje** [comment]
https://github.com/open-learning-exchange/planet/pull/2239#pullrequestreview-160201170

@dogi thanks for the feedback, I'll update soon.

**Comment by empeje** [comment]
This is for the COC @dogi https://github.com/open-learning-exchange/planet/pull/2245

**Comment by empeje** [comment]
When do you have time to review this @paulbert?

File: README.md [hunk 1]
```diff
@@ -1,84 +1,85 @@
...
+
+Similar as a nation to community, earth act as a publisher - perhaps the upstream publisher for all the content the community might consumes. It is exist as a way to exchange contents between nation.
+
+## Technologies
+
+We use Angular for our front-end and CouchDB as our back-end and database. And for the deployment we highly support Docker, but an old-school deployment in bare-metal or VM is also supported. We also support Raspberry-Pi deployment and in fact we have a lot of communities running in this little machine.
+
+## Getting Started
+
+1. Install CouchDB 2.0
```
Comment (dogi) [inline README.md]: hmm version should 2.2.0

File: README.md [hunk 2]
```diff
@@ -1,84 +1,85 @@
...
+
+    1.1. OSX
+
+        $ brew install couchdb
+
+    1.2. Ubuntu
+
+        $ apt-get install couchdb
```
Comment (dogi) [inline README.md]: does not work

File: README.md [hunk 3]
```diff
@@ -1,84 +1,85 @@
...
+We encourage you to contribute to Planet! Please check out the [Contributing to Planet](CONTRIBUTING.md) guide for guidelines about how to proceed. Join us!
+
+## License
+
+Planet is released under the [AGPL-3.0](LICENSE)
+
+[OLE]: https://www.ole.org/
+[OLE_CHATROOM]: https://gitter.im/open-learning-exchange/chat
+[MORE_GUIDELINE]: http://open-learning-exchange.github.io/#!./pages/vi/vi-planet-installation-and-configuration.md
+[COUCHDB_INSTALL]: http://docs.couchdb.org/en/latest/install/unix.html#installation-using-the-apache-couchdb-convenience-binary-packages
```
Comment (dogi) [inline README.md]: think this needs to move up to install instead of macOSX and Ubuntu section

Comment (empeje) [reply README.md]: it is only URL alias @dogi

File: README.md (hunk 1 above)
Comment (empeje) [reply README.md]: oops, will fix it

File: README.md (hunk 2 above)
Comment (empeje) [reply README.md]: oops, will fix it

File: README.md (hunk 3 above)
Comment (empeje) [reply README.md]: updated

Comment (empeje) [reply README.md]: how about this @dogi? have you check again?

File: README.md [hunk 4]
```diff
@@ -1,84 +1,85 @@
...
+
+        $ ./couchdb-setup.sh -p $COUCHDB_PORT -h $COUCHDB_DOMAIN -i
+
+    2.2.2 Next time (need to use the created credentials)
+
+        $ ./couchdb-setup.sh -p $COUCHDB_PORT -h $COUCHDB_DOMAIN -u $COUCHDB_USER -w $COUCHDB_PASS
+
+3. Run the app!
+
+        $ ng serve
```
Comment (paulbert) [inline README.md]: Missing the clone the repository & npm install steps.  For the simple install should we be suggesting a process which requires these steps, anyway?

File: README.md [hunk 5]
```diff
@@ -1,84 +1,85 @@
...
+
+4. Go to `localhost:4000` and you'll see 'Welcome to Planet'
```
Comment (paulbert) [inline README.md]: The default port is 3000 so this should be 3000 instead of 4000

File: README.md [hunk 6]
```diff
@@ -1,84 +1,85 @@
...
+
+        $ brew install couchdb
+
+    1.2. Ubuntu
+    
+    Follow the instruction in the official page [here][COUCHDB_INSTALL].
+
+    1.3. Docker (**Suggested!**)
+
+        $ docker run -p 5984:5984 -d --name my-couchdb treehouses/couchdb:2.2.0
```
Comment (paulbert) [inline README.md]: App defaults to db on port 2200, so we should use that.  Also we should put the suggested choice first on the list.

File: README.md [hunk 7]
```diff
@@ -1,84 +1,85 @@
...
+    2.1. Setup CORS in the CouchDB
+
+        $ npm install -g add-cors-to-couchdb
+        $ add-cors-to-couchdb $COUCHDB_HOST
+
+    2.2. Migrate schema and dummy data
+
+    2.2.1. First time installation
+
+        $ ./couchdb-setup.sh -p $COUCHDB_PORT -h $COUCHDB_DOMAIN -i
```
Comment (paulbert) [inline README.md]: We should just suggest `./couchdb-setup.sh -i` here and link to a section in contributing for more information about the options for this script

File: README.md [hunk 8]
```diff
@@ -1,84 +1,85 @@
...
+3. Run the app!
+
+        $ ng serve
+
+4. Go to `localhost:4000` and you'll see 'Welcome to Planet'
+5. Follow this [guidelines][MORE_GUIDELINE] to start using Planet in your community!
+
+## Contributing
+
+[![Open Source Helpers](https://www.codetriage.com/open-learning-exchange/planet/badges/users.svg)](https://www.codetriage.com/open-learning-exchange/planet)
```
Comment (paulbert) [inline README.md]: Also could you explain what codetriage.com is and why you're suggesting we include it here?

**Comment by paulbert** [comment]
Stale, closing

**Pull request closed without merging**